pkg/operators/kubeipresolver/kubeipresolver.go

```diff
@@ -49,8 +49,8 @@ const (
 type KubeNetworkInformation interface {
 	SetLocalPodDetails(owner, hostIP, podIP string, labels map[string]string)
 
-	GetRemoteIPs() []string
-	SetEndpointsDetails(endpoints []types.EndpointDetails)
+	GetEndpoints() []types.Endpoint
```

Comment:
We can have a single `GetEndpoints() []*types.Endpoint` method, by returning the pointer we allow the enricher to set its fields without having to use `SetEndpoints()` later on.

Reply:
IMO ^^^ that's too much overhead. I'd return the pointer and document what the operator fills.